ui/components/app/wallet-overview/eth-overview.test.js

```diff
@@ -471,4 +472,38 @@ describe('EthOverview', () => {
       );
     });
   });
+
+  describe('Disabled buttons when an account cannot sign transactions', () => {
+    const buttonTestCases = [
+      { testId: ETH_OVERVIEW_BUY, buttonText: 'Buy & Sell' },
```

Comment:
This case can be omitted

Reply:
Removed here [98cdbe0](https://github.com/MetaMask/metamask-extension/pull/23178/commits/98cdbe06be9592fb6c859a0ae2443f307f450296)